c_examples/mem-dump.c

```diff
@@ -24,7 +24,8 @@ void dump_memory(void* driver, const char* vm_name) {
     uint8_t buffer[PAGE_SIZE];
     for (int i = 0; i <= max_address / PAGE_SIZE; i++) {
         memset(buffer, 0, PAGE_SIZE);
-        if (microvmi_read_physical(driver, i * PAGE_SIZE, buffer, PAGE_SIZE)) {
+	    uint64_t bytes_read = 0;
```

Comment:
Indentation 😱 😱 😱

Reply:
Thanks.
I edited the file with vi in vagrant, and then with Intellij, and the IDE didn't complain about it.